dask_expr/_expr.py

```diff
@@ -2925,6 +2925,10 @@ def are_co_aligned(*exprs):
 ## Utilites for Expr fusion
 
 
+def is_valid_blockwise_op(expr):
+    return isinstance(expr, Blockwise) and not isinstance(expr, (FromPandas, FromArray))
```

Comment:
Wouldn't it be better to make `FromPandas` inherit from `IO` instead of `BlockwiseIO`?

Reply:
This would require us to patch other places (culling for example), we have to swallow the extra logic somewhere unfortunately